src/nfa/operations.cc

```diff
@@ -209,6 +223,7 @@ bool Mata::Nfa::make_complete(Nfa& aut, const Mata::Util::OrdVector<Symbol>& sym
 }
 
 //TODO: based on the comments inside, this function needs to be rewritten in a more optimal way.
+//Should it take an optional set of epsilons as a parameter?
```

Comment:
I think one could make use of a function such as this. I would support creating an overload for this function as you suggest.